compiler-plugin-test/src/test/resources/TestAggregateInLoop.kt

```diff
@@ -0,0 +1,13 @@
+import it.unibo.collektive.aggregate.api.Aggregate
+
+fun Aggregate<Int>.exampleAggregate(): Int = 0
```

Comment:
Can I suggest adding a test where you use the aggregate constructs (neighboring or exchange etc)? In order to have a more complete test.

Reply:
Ok, I've added a more flexible unit test in [161237f](https://github.com/Collektive/collektive/pull/454/commits/161237f864d9d37dc4846fd281ce243a8e3d21c1).
I've added a parametric line of code where it is possible to specify the aggregate function to put inside the loop.